packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -296,7 +296,12 @@ class OptimisticOracleContractMonitor {
       // Return the decoded ancillary data as a string. The `replace` syntax removes any escaped quotes from the string.
       return "Ancillary data: " + JSON.stringify(parseAncillaryData(ancillaryData)).replace(/"/g, "");
     } catch (_) {
-      return `Could not decode ancillary data: ${ancillaryData || "0x"}`;
+      try {
+        // If that fails, try to return the ancillary data UTF-8 decoded.
+        return "Ancillary could not be parsed. UTF-8 decoded data: " + this.web3.utils.hexToUtf8(ancillaryData);
```

Comment:
Thoughts on logging orig `ancillaryData` bytes as well as the decoded data? Would be long, but also mean that we wouldn't need to reference etherscan/other source to find this for proposals etc

Reply:
why? are you not going to do the same decoding? if this decoding fails then it logs the original data. is that not sufficient?

Reply:
I'm saying it's useful to log the original and decoded data at all times. Say someone wanted to use this log to manually propose or dispute a price, then it'd be useful to have original data on hand in the log.

Reply:
i think as long as you provide the txn hash you can grab the ancillary data easily from etherscan

Reply:
i'm indifferent. These alerts are infrequent enough that im not concerned about log message. So if @smb2796 does use this ancillary data then we should include

Reply:
Agreed! Raw ancillary data would be nice!

Reply:
ye I dont agree. I think that that mades these logs really hard to see, read and makes the slack channel hard to use. If you want the raw data go get it from etherscan. These messages end up being so large and unwieldily that the channel is just destroyed by massive messages

<img width="635" alt="image" src="https://user-images.githubusercontent.com/12886084/149286718-8b6b1d72-a662-426e-a180-cca5781ee3c5.png">


For almost everyone except perhaps one person it is better to let the bot decode it and if you want the raw data get it yourself.

do you agree?

Reply:
Also consider that slack has a max message length less than the max ancillary data! look at the screenshot above. in that case the message had to be redacted because ancillary data is so massive....

Reply:
Ok nbd. Fine without